dotcom-rendering/src/components/SlotBodyEnd.importable.tsx

```diff
@@ -194,6 +209,16 @@ export const SlotBodyEnd = ({
 				<SelectedEpic />
 			</div>
 		);
+	} else if (showArticleEndSlot) {
+		return (
+			<div id="slot-body-end">
+				<AdSlot
+					data-print-layout="hide"
+					position="article-end"
+					display={articleDisplay}
```

Comment:
I don't think you need to pass in `display`. Only the `right` ad slot depends on this.

Reply:
I will raise in a separate PR though I think